packages/cocoindex/hooks/proxy_supervisor.py

```
@@ -0,0 +1,349 @@
+                child_pid=self._child_pid,
+                inner_port=self.inner_port,
+                active_clients=self._active_clients,
+                last_disconnect_at="",
+                last_error="",
+            )
+            await self._apply_runtime_state()
+            if self._active_clients == 0:
+                self._schedule_idle_shutdown()
+            await self.shutdown_event.wait()
+            return 0
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)